Responsive-desktop/index.html

```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Week 2 Portifolio Website</title>
+  </head>
+  <body>
```

Comment:
- [ ] Let's try to fix your mobile version of the website. Remember that the idea behind working with `media` queries is that the styles from our mobile version remain intact while we modify the styles for a desktop screensize. The following screenshots show problematic areas of your site in the mobile version:

![image](https://user-images.githubusercontent.com/79114233/139560143-73058d36-ae71-4243-a2ca-422162655b9a.png)

![image](https://user-images.githubusercontent.com/79114233/139560146-0babd391-30f8-4b09-8b29-e2efb6a511a6.png)

![image](https://user-images.githubusercontent.com/79114233/139560152-cbc9fc7f-cf36-4d31-8b13-48852579abdd.png)

![image](https://user-images.githubusercontent.com/79114233/139560157-1d9eea94-9d3d-49bf-832d-f9e5c3c8982f.png)